bot/assets/filter_words.txt

```diff
@@ -0,0 +1,96 @@
+tranny
+\bfa+g(got)?
```

Comment:
Are these read as regexes?

Reply:
Yes, fairly obvious from code in `search.py` https://github.com/The-Codin-Hole/HotWired-Bot/blob/0a45e395fa0b8efab8548734d2ad5762de4b815c/bot/cogs/search.py#L12-L21